conf/fe.conf

```
@@ -24,10 +24,10 @@
 LOG_DIR = ${STARROCKS_HOME}/log
 
 DATE = "$(date +%Y%m%d-%H%M%S)"
-JAVA_OPTS="-Dlog4j2.formatMsgNoLookups=true -Xmx8192m -XX:+UseMembar -XX:SurvivorRatio=8 -XX:MaxTenuringThreshold=7 -XX:+PrintGCDateStamps -XX:+PrintGCDetails -XX:+UseConcMarkSweepGC -XX:+UseParNewGC -XX:+CMSClassUnloadingEnabled -XX:-CMSParallelRemarkEnabled -XX:CMSInitiatingOccupancyFraction=80 -XX:SoftRefLRUPolicyMSPerMB=0 -Xloggc:${LOG_DIR}/fe.gc.log.$DATE"
+JAVA_OPTS="-Dlog4j2.formatMsgNoLookups=true -Xmx8192m -Xms8192m -XX:+UseMembar -XX:+UseG1GC -XX:+HeapDumpOnOutOfMemoryErro -XX:+PrintGCDateStamps -XX:+PrintGCDetails -XX:SoftRefLRUPolicyMSPerMB=0 -Xloggc:${LOG_DIR}/fe.gc.log.$DATE"
```

HeapDumpOnOutOfMemoryErro
```suggestion
JAVA_OPTS="-Dlog4j2.formatMsgNoLookups=true -Xmx8192m -Xms8192m -XX:+UseMembar -XX:+UseG1GC -XX:+HeapDumpOnOutOfMemoryError -XX:+PrintGCDateStamps -XX:+PrintGCDetails -XX:SoftRefLRUPolicyMSPerMB=0 -Xloggc:${LOG_DIR}/fe.gc.log.$DATE"
```